static/bib/yoshitom.bib

```diff
@@ -1,3 +1,30 @@
+@misc{yoshitom-levorato-2020-icpr,
+  author = {Yoshitomo Matsubara and Marco Levorato},
+  title = {Neural Compression and Filtering for Edge-assisted Real-time Object Detection in Challenged Networks},
+  year = {2020},
+  archiveprefix = {arXiv},
+  code = {https://github.com/yoshitomo-matsubara/hnd-ghnd-object-detectors},
+  eprint = {2007.15818},
+  primaryclass = {cs.CV},
+}
+
+@misc{yoshitom-levorato-2020-emdl,
+  author = {Yoshitomo Matsubara and Marco Levorato},
+  title = {Split Computing for Complex Object Detectors: Challenges and Preliminary Results},
+  year = {2020},
+  archiveprefix = {arXiv},
+  code = {https://github.com/yoshitomo-matsubara/hnd-ghnd-object-detectors},
+  eprint = {2007.13312},
+}
+
+@inproceedings{yoshitom-vu-2020-sigir,
+  author = {Matsubara, Yoshitomo and Vu, Thuy and Moschitti, Alessandro},
+  title = {Reranking for Efficient Transformer-based Answer Selection},
+  booktitle = {Proceedings of the 43rd International ACM SIGIR Conference on Research and Development in Information Retrieval},
+  pages = {1577--1580},
+  year = {2020},
+}
```

Comment:
Since the paper has been already assigned a DOI, let us add it here to
enable the automatic Release link:

```suggestion
  doi = {10.1145/3397271.3401266},
}
```